frontend/src/patientApp/PatientHeader.tsx

```diff
@@ -36,6 +39,18 @@ const PatientHeader = () => {
             </div>
           </div>
         </div>
+        <div className="display-flex flex-align-end">
+          <Button
+            icon={"globe"}
+            className="usa-button--unstyled"
+            onClick={async () => {
```

Comment:
**nitpick:** this doesn't need to be `async`

Reply:
My intent here was to `await` the call to `i18n.changeLanguage` a few lines below since it returns a Promise (see https://github.com/CDCgov/prime-simplereport/pull/2186/commits/6c8a9fecb6086571d3e985ca3cbf4e49a1d3cec8#diff-4bd671efe13033c6548eb511d013b829081ed354ae1b5766d574862b7ad11b87R48) but I don't actually know if it makes a difference in this context?

Reply:
Yeah, there's no need to `await` it if you're not doing something later in the function that's dependent on being sure that operation is complete